Feature/meta data (#88)
DKE-Data/agrirouter-interface-documentation: docs/message-exchange.adoc

=== The default group

=== The default groups

==== The CU and machine group 

> Sometimes, new machines are added to the ecosystem, because a  Cu is connected, a (Virtual) CU is connected to a new machine or a Virtual CU is created by a telemetry platform. If this happens, a new endpoint for that machine is created in the end users agrirouter account. As the end user is not always logged in to setup routings, he can just setup routings for this â€“ not deletable â€“ default group. Every new endpoint is automatically assigned to the default group. Any endpoint can be removed from that group, the default group itself however cannot be deleted.

Reviewer: ```suggestion
Sometimes new machines are added to the ecosystem, because a CU is connected, a (virtual) CU is connected to a new machine or a virtual CU is created by a telemetry platform. If this happens, a new endpoint for that machine is created in the end users agrirouter account. As the end user is not always logged in to setup routings, he can just setup routings for a “not deletable“ default group. Every new endpoint is automatically assigned to the default group and can be removed from that group. The default group itself however cannot be deleted.
```
